test/tests/functional/pbs_cray_smoketest.py

```diff
@@ -54,12 +53,16 @@ def setUp(self):
         TestFunctional.setUp(self)
 
     @tags('cray', 'smoke')
-    def test_cray_apstat(self):
+    def test_cray_no_queued_job(self):
```

Comment:
I'm not sure you want to use the smoke tag.  This will cause them to be run by Travis.  They will correctly skip, but it will slow things down some.  They are smoke tests though, so they likely are correct having the tag.

Reply:
These were meant to be used in Cray smoke testing and tagged accordingly.